### ISSUE TITLE
rights problem on Ubuntu 22.04 for /tmp/zabbix-postfix-passive-statsfile.dat

### ISSUE THREAD

**Issue by belgotux** [rafael747/zabbix-postfix#15]
Hello,

Thanks for your plugin, work nice after a modification for a rights problem with passive script with the file `/tmp/zabbix-postfix-passive-statsfile.dat` on Ubuntu 22.04.
Same problem when I let the zabbix-server, or execute by `zabbix_agentd -t postfix[deferred]` or directly with `sudo /usr/local/sbin/zabbix_postfix_passive.sh deferred`.

![image](https://user-images.githubusercontent.com/11537542/205287929-ddb21177-7354-4be5-8f61-f121ad0c615a.png)
![image](https://user-images.githubusercontent.com/11537542/205288004-baad01a7-9726-428d-9dad-b0f6b5a30302.png)

I need to change the owner to root for working. And work like a charm. And something weird , when reapply owner and gorup to zabbix, it's continue to work... Until reboot and tmp files deleted of course.
![image](https://user-images.githubusercontent.com/11537542/205288070-f4cd3acd-2cce-462e-927d-ad8f7cb5456c.png)

In my case, no need to have zabbix as owner, it can be root like for `/tmp/zabbix-postfix-passive-offset.dat`, as we execute the script as root with the sudoers. I don't know why needed theses lines and I comment them.

I don't send a pull request, because maybe it's needed for something for someone ? But on Debian based I think not.

```
if [ ! -w "${PFSTATSFILE}" ]; then
        touch "${PFSTATSFILE}" && chown zabbix:zabbix "${PFSTATSFILE}" > /dev/null 2>&1

        if [ ! $? -eq 0 ]; then
                result_text="ERROR: wrong exit code returned while creating file ${PFSTATSFILE} and setting its owner to zaabbix:zabbix"
                result_code="1"
                write_result "${result_code}" "${result_text}"
        fi
fi
```

Have a nice day

**Comment by Lucarnosky** [comment]
Hi, I had the same problem and I've fixed it changing this line 
`touch "${PFSTATSFILE}" && chown zabbix:zabbix "${PFSTATSFILE}" > /dev/null 2>&1`
to this
`touch "${PFSTATSFILE}" > /dev/null 2>&1`

removing this it work without problem.

A workaround for this is to manually create the stats file in tmp folder

**Comment by rafael747** [comment]
I believe that @Lucarnosky's fix will do the job #16 
By letting the script owner create the file in `/tmp` we will not have permission issues while using it later

Thank you for the contribution
